backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/modelvalidator/M_ReceiptSchedule.java

```diff
@@ -37,13 +40,21 @@
 import org.adempiere.warehouse.validationrule.FilterWarehouseByDocTypeValidationRule;
 import org.compiere.model.I_C_OrderLine;
 import org.compiere.model.ModelValidator;
+import org.springframework.stereotype.Component;
 
 import java.math.BigDecimal;
 
 @Interceptor(I_M_ReceiptSchedule.class)
+@Component
```

Comment:
i think its fine, but i just want to make sure u checked somehow to not get this interceptor registered twice, so code will be called twice...

Reply:
It was called here https://github.com/metasfresh/metasfresh/pull/14010/files#diff-4834f924890ccfab5af8331a22402c710a90fc503ba1a83ca54a0b3b1a63b34a . But I deleted it.